chaste_codegen/cvode_with_data_clamp_model.py

```diff
@@ -0,0 +1,81 @@
+import sympy as sp
+
+from chaste_codegen.cvode_chaste_model import CvodeChasteModel
+
+
+class CvodeWithDataClampeModel(CvodeChasteModel):
+    """ Holds template and information specific for the CVODE with data clamp model type"""
+
+    def __init__(self, model, file_name, **kwargs):
+        super().__init__(model, file_name, **kwargs)
+        self._hpp_template = 'cvode_with_data_clamp_model.hpp'
+        self._cpp_template = 'cvode_with_data_clamp_model.cpp'
+
+    def _get_modifiable_parameters(self):
+        """ Get all modifiable parameters and adds membrane_data_clamp_current_conductance"""
+        modifiable_parameters = super()._get_modifiable_parameters()
+
+        # add membrane_data_clamp_current_conductance
+        self._membrane_data_clamp_current_conductance = \
```

Comment:
I did try that but I can't do that but I end up re-writing half of __ini__ if I do that since I need some things that are set in the super().__init__ to be able to do this.